File: leetCode/python/1st.py
class Solution:
    def toSum (self,nums:list[int],target:int)->list[int]:
        for i in range(len(nums)):
            for j in range(i+1,len(nums)):
                if(nums[i]+nums[j]==target):
                    return i,j
                

obj=Solution()

index=obj.toSum([2,7,9,1],10)
print("Index are: ",index)

File: leetCode/js/1st.js
const toSum=function(arr,target){
    for(let i=0;i<arr.length;i++){
        for(let j=i+1;j<arr.length;j++){
            if(arr[i]+arr[j]==target){
                console.log(`Sum of  ${arr[i]} and ${arr[j]} is :${arr[i]+arr[j]} which is eqaul to target value ${target}`)
                return [i,j]
            }
        }
    }

}

console.log(toSum([2,3,4,5],8))
